roles/openshift_logging_fluentd/tasks/main.yaml

```diff
@@ -23,6 +23,14 @@
     msg: openshift_hosted_logging_use_journal is deprecated.  Fluentd will automatically detect which logging driver is being used.
   when: openshift_hosted_logging_use_journal is defined
 
+- fail:
+    msg: Invalid openshift_logging_mux_client_mode [{{ openshift_logging_mux_client_mode }}], one of {{ __allowed_mux_client_modes }} allowed
+  when: openshift_logging_mux_client_mode is defined and not openshift_logging_mux_client_mode in __allowed_mux_client_modes
+
+- debug:
```

Comment:
@ewolinetz does assert abort the run?  I need two different behaviors

- If the user has defined `openshift_logging_mux_client_mode` and the value is not valid, I want to abort the ansible play with an error message
- If the user has defined `openshift_logging_mux_client_mode` and the value is `minimal`, I want to show a warning message, but continue the play

can I do that with `assert`?  If so, how?